offsets.py

```diff
@@ -98,6 +99,100 @@ class OffsetsFirmware_900:
     JMP_R14 = 0xffffffff82b85693
 
 
+# FW 9.03
+class OffsetsFirmware_903:
+    PPPOE_SOFTC_LIST = 0xffffffff843e99f8
+
+    KERNEL_MAP = 0xffffffff84464d48
+    SETIDT = 0xffffffff825128e0
+
+    KMEM_ALLOC = 0xffffffff8257a070
+    KMEM_ALLOC_PATCH1 = 0xffffffff8257a13c
+    KMEM_ALLOC_PATCH2 = 0xffffffff8257a144
+
+    MEMCPY = 0xffffffff82471130
+
+    # mov cr0 rsi ; ud2 ; mov eax 1; ret
+    MOV_CR0_RSI_UD2_MOV_EAX_1_RET = 0xFFFFFFFF823FB679 # 0F 22 C6 48 F7 C6
+
+    SECOND_GADGET_OFF = 0x3D
+
+    # jmp qword ptr [rsi + 0x3d]
```

Comment:
Use ROPgadget, put gadget in comment and use the address in lower case.

Reply:
> put gadget in comment

Instruction unclear, what do you mean here?

Reply:
Like 0xffffffff82996603 : jmp qword ptr [rsi + 0x3d] instead of jmp qword ptr [rsi + 0x3d]